src/headers.ts

```diff
@@ -458,6 +454,236 @@ export const inchi_StereoParity0D = new Enum({
+  lists_sterac: refNAPI.refType(refNAPI.refType(refNAPI.types.int)),
+});
+
+/* Input data structure for GetINCHIEx() */
+
+// typedef struct inchi_InputEx
+// {
+//     /* the caller is responsible for the data allocation and deallocation                           */
+
+//     /* same as in older inchi_Input                                                                 */
+//     inchi_Atom *atom;                       /* array of num_atoms elements                          */
+//     /* same as in older inchi_Input                                                                 */
+//     inchi_Stereo0D *stereo0D;               /* array of num_stereo0D 0D stereo elements or NULL     */
+//     /* same as in older inchi_Input                                                                 */
+//     char *szOptions;                        /* InChI options: space-delimited; each is preceded by  */
+//                                             /* '/' or '-' depending on OS and compiler              */
+//     /* same as in older inchi_Input                                                                 */
+//     AT_NUM num_atoms;                       /* number of atoms in the structure                     */
+//     /* same as in older inchi_Input                                                                 */
+//     AT_NUM num_stereo0D;                    /* number of 0D stereo elements                         */
+//     inchi_Input_Polymer *polymer;           /* v. 1.05+ extended data, polymers                      */
+//                                             /* NULL if not a polymer                                */
+//     inchi_Input_V3000 *v3000;               /* v. 1.05+ extended data, V3000 Molfile features        */
+//                                             /* NULL if no V3000 extensions present                  */
+// } inchi_InputEx;
+
+export const inchi_InputEx = NAPIStructType({
+  atom: refNAPI.refType(inchi_Atom),
+  Stereo0D: refNAPI.refType(inchi_Stereo0D),
+  szOptions: refNAPI.refType(refNAPI.types.char),
+  num_atoms: refNAPI.types.short,
+  num_stereo0D: refNAPI.types.short,
+  polymer: refNAPI.refType(inchi_Input_Polymer),
+  v3000: refNAPI.refType(inchi_Input_V3000),
+});
+
+/*
+    InChI -> Structure
+
+    GetStructFromINCHI()
+    GetStructFromStdINCHI()
+    GetStructFromINCHIEx()
+*/
+
+// typedef struct tagINCHI_InputINCHI
+// {
+//     /* the caller is responsible for the data allocation and deallocation       */
+//     char *szInChI;      /* InChI ASCIIZ string to be converted to a strucure    */
+//     char *szOptions;    /* InChI options: space-delimited; each is preceded by  */
+//                         /* '/' or '-' depending on OS and compiler */
+// } inchi_InputINCHI;
+
+export const inchi_InputINCHI = NAPIStructType({
+  szInChI: refNAPI.refType(refNAPI.types.char),
+  szOptions: refNAPI.refType(refNAPI.types.char),
+});
+=======
+/*************************************************
```

Comment:
Merge conflict is not properly resolved. The diff should only show the changes that you **added**. It should not undo the code that is already present in the `main` branch

Reply:
okay sir